tests/Export/XmlProductTest.php

```diff
@@ -159,15 +160,34 @@ public function testAttributeException(): void
             ->with(FindologicProductFactory::class)
             ->willReturn($findologicFactoryMock);
 
-        $xmlProduct = new XmlProduct(
+        $xmlProduct = $this->getDefaultXmlProduct($productEntity, $containerMock);
+        $xmlProduct->buildXmlItem();
+    }
+
+    public function testKeywordsAreNotRequired(): void
+    {
+        $product = $this->createVisibleTestProduct(['tags' => []]);
+
+        $xmlProduct = $this->getDefaultXmlProduct($product);
+        $xmlProduct->buildXmlItem();
+
+        $xmlItem = $xmlProduct->getXmlItem();
+
+        $this->assertNotNull($xmlItem);
+        $this->assertSame($product->getId(), $xmlItem->getId());
```

Comment:
Please also assert that the keywords are empty.

Reply:
See [this comment](https://github.com/findologic/plugin-shopware-6/pull/124/files#r520412819).